app/views/components/_contents-list.html.erb

```diff
@@ -11,7 +12,13 @@
                               end
 %>
 <% if contents.any? %>
-  <nav role="navigation" class="app-c-contents-list">
+  <nav
+    role="navigation"
+    class="app-c-contents-list"
+    <% unless aria_label.blank? %>
+      aria-label="<%= aria_label %>"
```

Comment:
I wonder if the component should have a default, eg "Page contents list"

Reply:
I decided against putting a default in since we've not been adding anything previously and it makes sense to be able to decide your context at the time of use. If somebody using it for cross-page contents lists forgets to supply a label then that particular default text seems misleading. Are you really keen for this to be added?